components/sharing/SocialShareModal.stories.js

```diff
@@ -0,0 +1,43 @@
+import { storiesOf } from '@storybook/vue';
+import Vuex from 'vuex';
+import SocialShareModal from './SocialShareModal.vue';
+import ShareButton from './ShareButton.vue';
+
+const i18n = {
+  locale: 'en',
+  messages: {
+    en: {
+      actions: {
+        close: 'close',
+        share: 'share',
+        "shareOn": "Share on {social}"
+      }
+    }
+  }
+};
+
+const store = new Vuex.Store({
+  getters: {
+    'http/canonicalUrl': () => {}
```

Comment:
return a value here 'https://www.example.org/'

Reply:
as in `'http/canonicalUrl': () => { 'https://www.example.org/' }` ?